Resources/Prototypes/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```
@@ -210,8 +210,8 @@
 - type: entity
   name: N1984
   parent: [BaseWeaponPistol, BaseSecurityContraband]
-  id: WeaponPistolN1984 # the spaces in description are for formatting.
-  description: The sidearm of any self respecting officer.     Comes in .45 magnum, the lord's caliber.
+  id: WeaponPistolN1984
+  description: An exceptionally powerful ‘hand cannon’ manufactured by a short-lived Nanotrasen subsidiary. It was passed over by most buyers due its hefty price and even heftier recoil, but has become something of a status symbol for NT officials. Chambered in .45, the lord’s caliber. It feeds from .45 pistol magazines.
```

Nitpick:
This description is massive, and I have some cosmetic issues:
- Nanotrasen does not really do subsidiaries, they do divisions, ex. Small Arms Division. Remember they're supposed to be a monolithic corporation (rather than a secretive monopoly) it's all one company.
- Feeding from .45 magazines is redundant because we already state that it's chambered in .45
- The stuff about buyers is a lot of filler text, but I like the status symbol part.


A possible suggestion is:
```suggestion
  description: An exceptionally powerful ‘hand cannon’ manufactured by Nanotrasen's BFG division. An iconic firearm, serving as a status symbol for NT officials. Chambered in .45, the lord’s caliber.
```

---

I really like the “BFG division” and that new description. Will update.